python-frontend/src/test/java/org/sonar/python/semantic/v2/TypeInferenceV2Test.java

```diff
@@ -362,12 +360,9 @@ def foo(self):
           ...
       """
     );
-    Optional<ClassSymbol> classA = fileInput.globalVariables().stream().filter(s -> s.name().equals("A")).map(ClassSymbol.class::cast).findFirst();
-    assertThat(classA).isPresent();
-    // SONARPY-2327 The method call to foo() in class A is considered as a member of A, while if A was defined in another file than B, it won't be.
-    assertThat(classA.get().canHaveMember("foo")).isTrue();
-    assertThat(classA.get().declaredMembers()).extracting("kind", "name")
-      .containsExactlyInAnyOrder(Tuple.tuple(Symbol.Kind.FUNCTION, "meth"), Tuple.tuple(Symbol.Kind.OTHER, "foo"));
+    ClassType classA = (ClassType) ((ClassDef) fileInput.statements().statements().get(0)).name().typeV2();
+    assertThat(classA.members().stream().anyMatch(m -> m.name().equals("foo"))).isFalse();
+    assertThat(classA.members().stream().anyMatch(m -> m.name().equals("meth"))).isTrue();
```

Comment:
Using the V2 type inference functions, `foo` is no longer a member of class A. Thus, I removed the comment referring to ticket SONARPY-2327: It is not a problem of inconsistency between declaring the parent class and the child class in the same file or in two separate files, but a problem of migrating the rule to the V2 type. 
If you agree, I'll close the ticket 2327. WDYT?